snappy/common_test.go

```diff
@@ -121,19 +136,37 @@ func storeMinimalRemoteManifest(qn, name, origin, version, desc, channel string)
 	return nil
 }
 
-func addDefaultApparmorJSON(tempdir, apparmorJSONPath string) error {
-	appArmorDir := filepath.Join(tempdir, "var", "lib", "apparmor", "clicks")
+func addMockDefaultApparmorProfile(appid string) error {
+	appArmorDir := dirs.SnapAppArmorDir
+
 	if err := os.MkdirAll(appArmorDir, 0775); err != nil {
 		return err
 	}
 
-	const securityJSON = `{
-  "policy_vendor": "ubuntu-core"
-  "policy_version": 15.04
+	const securityProfile = `
+#include <tunables/global>
+profile "foo" (attach_disconnected) {
+	#include <abstractions/base>
 }`
 
-	apparmorFile := filepath.Join(appArmorDir, apparmorJSONPath)
-	return ioutil.WriteFile(apparmorFile, []byte(securityJSON), 0644)
+	apparmorFile := filepath.Join(appArmorDir, appid)
+	return ioutil.WriteFile(apparmorFile, []byte(securityProfile), 0644)
+}
+
+func addMockDefaultSeccompProfile(appid string) error {
+	seccompDir := dirs.SnapSeccompDir
+
+	if err := os.MkdirAll(seccompDir, 0775); err != nil {
+		return err
+	}
+
+	const securityProfile = `open
+write
+connect
+`
```

Comment:
FWIW, I generally try to avoid formatting strings like this, and give preference to one of these because they are seemingly more readable as they hold the content aligned:

```
const s = `
a
b
c
`

func f() {
        const s = "a\nb\nc"

        const s = "" +
                "a\n" +
                "b\n" +
                "c\n"
)